types/invariant/invariant-tests.ts

```diff
@@ -21,5 +21,5 @@ invariant('', 'Some other error');
 // handles extra variables
 invariant(true, 'Error, error, read all about it', 37, {}, 'hello');
 
-// $ExpectType {}
+// $ExpectType void | {}
```

Comment:
I believe this should stay as `{}` to facilitate this pattern:

```ts
const thing = getThing() || invariant(false, 'Expected getThing to return something')
```

as well as

```ts
const thing = getThing() ?? invariant(false, 'Expected getThing to return something')
```